pkg/operator/ceph/object/spec.go

```diff
@@ -69,21 +68,29 @@ func (c *clusterConfig) createDeployment(rgwConfig *rgwConfig) (*apps.Deployment
 }
 
 func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) (v1.PodTemplateSpec, error) {
+	shareProcessNamespace := true
 	podSpec := v1.PodSpec{
 		InitContainers: []v1.Container{
 			c.makeChownInitContainer(rgwConfig),
 		},
-		Containers: []v1.Container{
-			c.makeDaemonContainer(rgwConfig),
-		},
+		Containers:    []v1.Container{c.makeDaemonContainer(rgwConfig)},
 		RestartPolicy: v1.RestartPolicyAlways,
 		Volumes: append(
 			controller.DaemonVolumes(c.DataPathMap, rgwConfig.ResourceName),
 			c.mimeTypesVolume(),
 		),
-		HostNetwork:       c.clusterSpec.Network.IsHost(),
-		PriorityClassName: c.store.Spec.Gateway.PriorityClassName,
+		HostNetwork:           c.clusterSpec.Network.IsHost(),
+		PriorityClassName:     c.store.Spec.Gateway.PriorityClassName,
+		ShareProcessNamespace: &shareProcessNamespace,
```

Comment:
Why was it that we need the shared pid space?

Reply:
We need to share PID space so that we can send a HUP signal to the main Ceph daemon process to reload and rotage logs.

Reply:
This only shares PID namespace within the Pod right? Not host-wide?

Reply:
Yes just across the pods, **not** HostPID.